src/docs/team.md

```diff
@@ -0,0 +1,35 @@
+The membership of Django Commons can be viewed on GitHub here: TODO
+
+## Current Admins
+
+Django Commons is currently administered by the following people. If you'd like to join them, ... TODO
```

Comment:
Of course, I mark it ready for review and immediately see "TODO" in the text :melting_face:

Reply:
```suggestion
Django Commons is currently administered by the following people. If you'd like to join them, please open a "New admin" issue at [django-commons/membership](https://github.com/django-commons/membership/issues).
```